tests/test_other.py

```diff
@@ -6642,10 +6642,9 @@ def test_metadce_cxx(self, *args):
     'export_nothing':
           (['-Os', '-s', 'EXPORTED_FUNCTIONS=[]'],    [], [],     43), # noqa
     # we don't metadce with linkable code! other modules may want stuff
-    # don't compare the # of functions in a main module, which changes a lot
```

Comment:
is the idea to restore this after finishing the ABI change, that is, to see the differences, but then stop looking for them? or will this be useful in the long term?

Reply:
They useful long term .. i think this comment was for MAIN_MODULE=1.. for MAIN_MODULE=2 the list is very small and we do want to know when it changes.

Reply:
Its only 20 functions

Reply:
And they won't change very often all right?

Reply:
They may change in response to codegen changes in LLVM or optimizations in binaryen - inlining can make a function "disappear", and a small change can cross the inlining threshold. But yes, maybe mode 2 is less risky than mode 1, we'll see...

Reply:
True, but I don't think the MODULE_MODULE=2 test is any more or less likely to change than the existing metadce tests is it?

I think the MODULE_MODULE=1 was the one that was more fragile since it included all symbols.

Reply:
Maybe not much, yeah. Looks like we already have tests with more than 20 functions.